UPGRADE-3.0.md

```diff
@@ -4,6 +4,10 @@ UPGRADE FROM 2.x to 3.0
 Configuration
 -------------
 
+### The `collate` default table option is removed
```

Comment:
Can you expand the full path of the options please. Here I think it's `doctrine.dbal.default_table_options.collate`